app/frontend/src/features/messages/Messages.tsx

```diff
@@ -0,0 +1,23 @@
+import { TabContext, TabPanel } from "@material-ui/lab";
+import * as React from "react";
+import PageTitle from "../../components/PageTitle";
+import TabBar from "./TabBar";
+import { labels } from "./constants";
+
+export default function Messages() {
+  const [value, setValue] = React.useState<keyof typeof labels>("TAB_ALL");
+  return (
+    <>
+      <PageTitle>Messages</PageTitle>
+      <TabContext value={value}>
+        <TabBar value={value} setValue={setValue} labels={labels} />
+        <TabPanel value={"TAB_ALL"}>ALL</TabPanel>
+        <TabPanel value={"TAB_GROUPCHATS"}>GROUPCHATS</TabPanel>
+        <TabPanel value={"TAB_HOSTING"}>HOSTING</TabPanel>
+        <TabPanel value={"TAB_SURFING"}>SURFING</TabPanel>
+        <TabPanel value={"TAB_MEET"}>MEET</TabPanel>
+        <TabPanel value={"TAB_ARCHIVED"}>ARCHIVED</TabPanel>
```

Comment:
~~Group chats with a space. Also I think this should be sentence case and use css transforms for all caps (or just sentence case for now).~~ Nevermind these are just the placeholders

Reply:
But I agree with the idea to not use layout in text.